test/e2e/network_segmentation_services.go

```diff
@@ -256,19 +256,33 @@ var _ = Describe("Network Segmentation: services", func() {
 // TODO Once https://github.com/ovn-org/ovn-kubernetes/pull/4567 merges, use the vendored *TestJig.Run(), which tests
 // the reachability of a service through its name and through its cluster IP. For now only test the cluster IP.
 
-const OVNNodeHostCIDRs = "k8s.ovn.org/host-cidrs"
+const OvnNodeIfAddr = "k8s.ovn.org/node-primary-ifaddr"
```

Comment:
for "udn, e2e: Don't use secondary node IPs for node port test." it would be good to have a commit message explaining why this change is needed